admin/links/class-link-watcher.php

```diff
@@ -39,6 +39,10 @@ public function register_hooks() {
 	 * @return void
 	 */
 	public function save_post( $post_id, WP_Post $post ) {
+		if ( ! WPSEO_Link_Table_Accessible::check_table_is_accessible() || ! WPSEO_Meta_Table_Accessible::is_accessible() ) {
```

Comment:
We are now calling `check_table_is_accessible` instead of `is_accessible` on the Link Table.
This does not seem like the most efficient call.

Creating a new issue to determine if the accessibility check can be improved up.